storybook/main.cpp

```diff
@@ -50,13 +55,13 @@ int main(int argc, char *argv[])
     qmlRegisterSingletonType<CacheCleaner>(
                 "Storybook", 1, 0, "CacheCleaner", cleanerFactory);
 
-    const QUrl url(SRC_DIR + QStringLiteral("/main.qml"));
+    const QUrl url(QML_IMPORT_ROOT + QStringLiteral("/main.qml"));
     QObject::connect(&engine, &QQmlApplicationEngine::objectCreated,
                      &app, [url](QObject *obj, const QUrl &objUrl) {
         if (!obj && url == objUrl)
             QCoreApplication::exit(-1);
     }, Qt::QueuedConnection);
     engine.load(url);
 
-    return QGuiApplication::exec();
+    return app.exec();
```

Comment:
No, it's not related to wasm and you are right, app.exec() is not needed in our case.
This is usually used when you don't know what type of app you're running, but it's not the case here. Will update.

Reply:
In case of static invocation the type is always known. The only benefit is that we don't have to change type in invocation when changing type of app. But if it's the case I would consider `decltype(app)::exec();` to underline that its static call. But it looks quite fancy.